backend/grants/summary.py

```diff
@@ -0,0 +1,119 @@
+from django.db.models import Count, Sum
+from conferences.models.conference import Conference
+from helpers.constants import GENDERS
+from countries import countries
+from .models import Grant
+
+
+class GrantSummary:
```

Comment:
it might be good to cover this class with tests if we don't have them already? how do we make sure the calculations are correct?